src/js/modules/infragistics.ui.scroll.js

```diff
@@ -754,6 +754,8 @@
 			this._linkedVBar = null;
 			this._elemWidth = elem.width();
 			this._elemHeight = elem.height();
+			this._minVDragHeightPx = 15; // in px
```

Comment:
Shouldn't this be a setting with a default value of `15`?

Reply:
We decided to not add it as a new option yet but it can easily be exposed.

Reply:
@skrustev It's better to expose it, otherwise this is a "magic number".

Reply:
@kdinev I've exposed it as an option.